docs/Building.md

```diff
@@ -0,0 +1,112 @@
+#How to Build Open Network Linux 
+
+
+Build Hosts and Environments
+------------------------------------------------------------
+ONL now builds with Docker (yay!) so the only requirements on the
+build system is:
+
+- docker			# to grab the build workspace
+- binfmt-support		# kernel support for ppc builds
+
+On a modern Ubuntu system, you can get these with:
+
+    # apt-get install docker.io binfmt-support
+
+Historical Note: the previous workspace workflow (with `onl-mkws` and
+`onl-chws`) was actually a home grown Docker-like Linux container based
+system, so this is not a fundamental change in how ONL is built.
+
+
+Build ONL Summary
+------------------------------------------------------------
+    #> git clone git://github.com/opennetworklinux/ONL
+    #> cd ONL
+    #> make docker                                              # enter the docker workspace
+    root@8onl-builder:/path/to/ONL# make onl-x86 onl-powerpc    # build both x86 and PPC images
+
+The resulting ONIE installers are in $ONL/builds/installer/$ARCH/all/onl-$VERSION-all.installer,
+and the SWI files (if you want them) are in $ONL/builds/swi/$ARCH/all/onl-$VERSION-all.swi.
+
+
+
+Additional Details
+----------------------------------------------------------
+
+The rest of this guide talks about how to build specific 
+sub-components of the ONL ecosystem and tries to overview
+all of the various elements of the build.
+
+Installing Docker Gotchas
+----------------------------------------------------------
+Docker installer oneliner (for reference: see docker.com for details)
+
+    # apt-get install -y docker.io
+or
+
+    # wget -qO- https://get.docker.com/ | sh
+
+
+If you are installing on Ubuntu 14.04 or earlier:
+
+- You may have to update your kernel to 3.10+
+- Beware that `apt-get install docker` installs a dock application not docker :-)  You want the docker.io package instead.
+- Check out http://docs.docker.com/installation/ubuntulinux/ for details
+
+Consider enabling builds for non-priviledged users with:
```

Comment:
Fixed.  Appreciate the diplomacy of "typo" versus "Rob is a fool who can't spell"